src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -2347,5 +2347,27 @@ public boolean isVideo()
             String extension = getFileExtension();
             return extension != null && VIDEO_EXTENSIONS.contains(extension.toLowerCase());
         }
+
+        /**
+         * Whether or not this attachment is marked as spoiler,
+         * based on {@link #getFileName()}.
+         *
+         * @return True if this attachment is marked as spoiler
+         */
+        public boolean isSpoiler()
+        {
+            return getFileName().startsWith("SPOILER_");
+        }
+        
+        /**
+         * Whether or not this attachment is an image and marked as spoiler,
+         * based on {@link #isSpoiler()}, and {@link #isImage()}.
+         *
+         * @return True if this attachment is and image and marked as spoiler
+         */
+        public boolean isImageSpoiler()
```

Comment:
I don't get the point of this method. We don't need a helper method for this and this will just result in people asking "why is there no isVideoSpoiler and isTextSpoiler and isNotVideoSpoiler" which I just really don't wanna deal with that.